packages/cra-external-template/template/config-overrides.js

```diff
@@ -0,0 +1,53 @@
+const { override, addBabelPreset } = require('customize-cra')
```

Comment:
I think we need the new way to override config instead of do like this

Reply:
I update config to match react-rewire config convention. Check it out @tanphamhaiduong: https://github.com/reapit/foundations/pull/2068/files/c80ac9a5a6df65157a0864038736cefae02df916#diff-4d5bed7adf629fc59b9f0795879b3b8e